core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -149,6 +157,15 @@ private[data] sealed trait WriterTMonad[F[_], L] extends WriterTApplicative[F, L
     fa.flatMap(f)
 }
 
+private[data] sealed trait WriterTMonadError[F[_], L, E] extends MonadError[WriterT[F, L, ?], E] with WriterTMonad[F, L] {
+  override implicit def F0: MonadError[F, E]
+
+  def raiseError[A](e: E): WriterT[F, L, A] = WriterT(F0.raiseError[(L, A)](e))
+
+  def handleErrorWith[A](fa: WriterT[F, L, A])(f: E => WriterT[F, L, A]): WriterT[F, L, A] =
+    WriterT(F0.handleErrorWith(fa.run)(e => f(e).run))
```

Comment:
I think you are right. I was thinking of something like XorT -> WriterT -> Future, but here F is just Future so there's no monadError instance for it. XorT will provide the MonadError and will already do the right thing.